src/test/java/nextstep/courses/domain/session/EnrollmentsTest.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.courses.domain.session;
+
+import nextstep.courses.CannotEnrollException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static nextstep.courses.domain.course.CourseTest.COURSE;
+import static nextstep.courses.domain.coverImage.CoverImageTest.COVER_IMAGE_PNG;
+import static nextstep.courses.domain.session.PeriodTest.PERIOD_OF_SESSION;
+import static nextstep.courses.domain.session.SessionTypeTest.PAID_SESSION_TYPE;
+import static nextstep.users.domain.NsUserTest.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class EnrollmentsTest {
+    private Session session = new Session(11L, "유료 강의", "유료 강의다", PAID_SESSION_TYPE, PERIOD_OF_SESSION, new CoverImages(List.of(COVER_IMAGE_PNG)), COURSE);
+
+    @Test
+    @DisplayName("numberOfCurrentEnrollment(): 현재 수강 승인된 수강자들의 수를 반환한다.")
+    void testNumberOfCurrentEnrollment() {
+        Enrollment enrollmentOfSANJIGI = new Enrollment(session, SANJIGI);
+        Enrollment enrollmentOfZIPJIGI = new Enrollment(session, ZIPJIGI);
+        enrollmentOfZIPJIGI.approve(JAVAJIGI);
+
+        Enrollments enrollments = new Enrollments(List.of(enrollmentOfSANJIGI, enrollmentOfZIPJIGI));
+
+        assertThat(enrollments.numberOfCurrentEnrollment()).isEqualTo(1);
```

Comment:
```suggestion
        Enrollment approved = new Enrollment(session, SANJIGI, APPROVED);
        Enrollment notApproved = new Enrollment(session, ZIPJIGI);
        Enrollments enrollments = new Enrollments(List.of(approved, notApproved));

        long actual = enrollments.numberOfCurrentEnrollment();

        assertThat(actual).isEqualTo(1);
```

`enrollmentOfSANJIGI`  테스트에서 `SANJIGI`의 의미가 중요할까요?
테스트 코드를 읽고 어떠한 테스트인지 빠르게 인지할수 있도록 하는게 더 중요하지 않을까요?

변수가 해당 스코프 영역안에서 어떠한 의미를 가지는지 표현한다면 어떨까요?

저는 `long actual = enrollments.numberOfCurrentEnrollment();` 이렇게 표현해볼것 같아요.
그리고 누군가 테스트 코드를 보면, `아! numberOfCurrentEnrollment 이걸 테스트 하려고하는구나!` 빠르게 이해할수 있다고 생각해요 😄

Reply:
제가 테스트 코드 작성시 의미전달 할 떄 너무 테스트 코드 메서드의 이름과 DisplayName마 신경쓴거 같습니다. 좋은 조언 감사합니다!